lib/email_alert_title_builder.rb

```diff
@@ -53,16 +42,20 @@ def plural_or_single
   end
 
   def prefix
-    if facets.empty?
-      subscription_list_title_prefix.to_s
-    elsif facets.size == 1
+    if facets.size == 1
       subscription_list_title_prefix[plural_or_single].to_s
     elsif selected_facets.empty?
       subscription_list_title_prefix.to_s
     elsif subscription_list_title_prefix
       "#{subscription_list_title_prefix}with "
```

Comment:
is there a missing space before `with`?
`"#{subscription_list_title_prefix}with "` vs `"#{subscription_list_title_prefix} with "`